workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequireTagInjector.cs

```diff
@@ -10,7 +10,7 @@ namespace Improbable.Gdk.Core.GameObjectRepresentation
     /// <summary>
     ///     Retrieves Reader and Writer fields from MonoBehaviours and handles injection into them.
     /// </summary>
-    internal class SpatialOSBehaviourLibrary
+    internal class RequireTagInjector
```

Comment:
As it doesn't inject require tags this name could be better although I think you get the basic idea from it.

Reply:
Can't really think of any more correct name that is not also annoyingly unclear/vague... :/

Reply:
`RequiredFieldInjector`
`RequiredFieldResolver`
`SpatialAccessorInjector` / `Resolver`
`RequireResolver`
`MonoBehaviourInjection`
`MonoBehaviourInjectionResolver`
`RequirementResolver`
`SpatialInjector`
`SpatialBehaviourInjector`